website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -145,43 +144,45 @@ Advanced users can configure multiple projects to use the same Cloud CLI executa
 
 <TabItem value="pip" label="Existing dbt Core users (pip)">
 
+If you already have dbt Core installed, the dbt Cloud CLI may conflict. Here are some considerations:
 
-:::info Use native packages or a virtual environment to prevent dbt Core conflicts
-
-To prevent overwriting dbt Core, avoid installing the dbt Cloud CLI with pip. Instead, consider using the native installation method and configuring your PATH or create a new virtual environment.
-
-If you've already installed the dbt Cloud CLI and need to switch back to dbt Core, uninstall the dbt Cloud CLI, and follow the dbt Core installation instructions.
-
-You can also have both dbt Cloud CLI and dbt Core installed simultaneously. To avoid conflicts, alias the dbt Cloud CLI as `dbt-cloud`. For more details, check the [FAQs](#faqs) if your operating system experiences path conflicts.
-:::
+- **Prevent conflicts** <br /> If you want to use `pip`, create a new virtual environment.<br /><br />
```

Comment:
I think "Use both dbt Cloud CLI and dbt Core with pip" is the better heading here.